### DISCUSSION TITLE
Nombre del Proyecto

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#25] [Ideas]
Creo esta discusion para tirar ideas para el nombre para el proyecto.

**Comment by BraianTroncoso** [comment]
Grido helados.

**Comment by rpgrca** [comment]
El cuatrimestre pasado fue BidOn, este cuatrimestre podríamos ser EnBalde, como un juego de palabras con Embalse de Córdoba.

**Comment by Tarias89** [comment]
• Bien Helados
• Heladomania

**Comment by Brian-Almada** [comment]
No te pasa que cuando estás estudiando despierta dentro tuyo unas ganas tremendas de comer algo rico. Bueno, nosotros tenemos la solución para vos. Helasity es el lugar en donde están los mejores helados de la ciudad. Entra en Helasity.com.ar y elegí el sabor que más te guste mientras seguís estudiando. 

Helasity! 
El sabor que va con vos mientras la rompes en la university ;)

**Comment by NataliaAlvarezIspc** [comment]
Voto por EnBalde!!!
